### ISSUE TITLE
Uniprot (flat file)

### ISSUE THREAD

**Issue by slobentanzer** [biocypher/biocypher#194]

**Comment by carissableker** [comment]
Adapter url seems to be broken, should be https://github.com/biocypher/igvf-catalog/blob/dev/data/adapters/uniprot_adapter.py?

**Comment by slobentanzer** [comment]
Yes, thanks; this board has fallen out of maintenance and we are currently redesigning the adapter registry from scratch.
